userservice/src/main/java/ro/devdepot/services/implementation/UserServiceImpl.java

```diff
@@ -27,4 +40,106 @@ public LoginResponse login(LoginRequest loginRequest) {
 
         return null;
     }
+
+    @Override
+    public String createUser(CreateUserRequest createUserRequest) {
+
+        validateUserDto(createUserRequest);
+
+        User userCreated = userRepository.save(userMapper.getUserFrom(createUserRequest));
+
+        if(userCreated.getId() != null) {
+            return "User created with successfully";
+        }
+        throw new BusinessException("Something goes wrong", HttpStatus.CONFLICT);
+    }
+
+    @Override
+    public String updateUser(UpdateUserRequest updateUserRequest, Long id) {
+
+        User user = userRepository.findById(id).orElseThrow(() -> new BusinessException("User with id: " + id + " was not found", HttpStatus.NOT_FOUND));
+        user.setEmail(updateUserRequest.getEmail());
+        user.setFirstName(updateUserRequest.getFirstName());
+        user.setLastName(updateUserRequest.getLastName());
+        userRepository.save(user);
+
+        return "User " + user.getUsername() + " has been saved successfully";
```

Comment:
Some here, you can put the message in the controller, here either make the method void or return the user or user id